app/controllers/concerns/nsm/adjustment_concern.rb

```diff
@@ -1,11 +1,21 @@
 module Nsm
   module AdjustmentConcern
     extend ActiveSupport::Concern
+    # rubocop:disable Style/NumberedParameters
     def confirm_deletion
       authorize(claim, :update?)
-      @adjustment = adjustments.find { _1.id == params[:id] }
+
+      @adjustment = if additional_fee?
+                      view_model
+                    else
+                      view_model.filter(&:any_adjustments?).find do
+                        _1.id == params[:id]
```

Comment:
I think Rubocop is right here, it's not much cleaner to use a numbered parameter

Reply:
true - I think when I initially ignored this it was all in one line without the do block